libnetwork/drivers/overlay/encryption.go

```diff
@@ -638,7 +638,7 @@ func clearEncryptionStates() {
 	}
 	for _, sa := range saList {
 		if sa.Reqid == r {
-			if err := nlh.XfrmStateDel(&sa); err != nil {
+			if err := nlh.XfrmStateDel(&sa); err != nil { // nolint:gosec
```

Comment:
Same here; this may be legit; we probably need to capture `sa` inside the loop (`sa := sa`)